master/pkg/searcher/custom_searcher_events_queue.go

```diff
@@ -35,7 +35,6 @@ func (q *SearcherEventQueue) GetEvents() []*experimentv1.SearcherEvent {
 func (q *SearcherEventQueue) RemoveUpTo(eventID int) error {
 	for i, v := range q.events {
 		if v.Id == int32(eventID) {
-			q.eventCount -= -v.Id
```

Comment:
We decided that q.eventCount is the total event count. It is incremented with each event and never decremented.